### DISCUSSION TITLE
New contributor: lakshayrohila

### DISCUSSION THREAD

**Discussion by dimkr** [puppylinux-woof-CE/woof-CE#3642] [Polls]
@puppylinux-woof-CE/founders @puppylinux-woof-CE/members

I think @lakshayrohila is doing good work: everybody in the forum is talking about developing a new Puppy version but nobody is doing anything. It's rare to have a new contributor, especially one with the willingness to fix issues and come up with improvements.

Therefore, I think it would be nice to add Lakshay to this organization with `Read` access. Members with `Read` access can comment on issues and review pull requests (see [this table](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization)), making it easier to collaborate with them compared to non-members. Later, as confidence grows, this can change to `Triage` or `Write`.

Opinions?

**Comment by lakshayrohila** [comment]
Thank you for considering me for this. Its my pleasure ☺️!
